index.html

```diff
@@ -0,0 +1,173 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Yayner Ecommerce</title>
+    <script src="https://use.fontawesome.com/75d401af0c.js"></script>
+    <link rel="stylesheet" href="assets/css/style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Barlow+Semi+Condensed:ital,wght@0,200;0,300;0,400;0,700;1,300&family=Lato:ital,wght@0,300;0,400;0,700;0,900;1,300;1,400;1,700;1,900&family=Poppins:ital,wght@0,300;0,400;0,600;1,200&family=Sora:wght@100;200;300;400;500;600;700;800&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <header class="header">
+      <div class="social-login">
+        <div class="container">
+          <ul class="socila-login-items">
+            <li>
+              <a href="#"><i class="fa fa-facebook" aria-hidden="true"></i></a>
+            </li>
+            <li>
+              <a href="#"><i class="fa fa-twitter" aria-hidden="true"></i></a>
+            </li>
+            <li>
+              <a href="#"
+                ><i class="fa fa-envelope-o" aria-hidden="true"></i
+              ></a>
+            </li>
+            <li><a href="#">Amharic</a></li>
+            <li><a href="#">Login</a></li>
+          </ul>
+        </div>
+      </div>
+      <nav class="nav-bar">
+        <div class="container">
+          <div class="logo">
+            <a href="index.html"><img
+              src="assets/images/yaycommerceNobg.png"
+              alt="logo"
+              class="logo"
+            /></a>
+          </div>
+          <ul class="navigation">
+            <li><a href="index.html" class="home">Home</a></li>
+            <li><a href="about.html">About</a></li>
+            <li><a href="#">Join</a></li>
+            <li><a href="#">Sponsor</a></li>
+            <li><a href="#">Blog</a></li>
+            <li><a href="#" class="contact">Buy Now</a></li>
+          </ul>
+        </div>
+      </nav>
+    </header>
+    <main class="main">
+      <div class="container">
+        <div class="toggle"></div>
+      </div>
+      <div class="container">
+        <section class="main-title">
+          <h1><q>Wellcome to YayCommerce</q></h1>
+          <h2>
+            Ethiopia's Largest <br />
+            E-commerce Site
+          </h2>
+        </section>
+        <div class="main-description">
+          <p>
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Architecto
+            quidem consequatur laboriosam accusantium a eveniet accusamus dicta
+          </p>
+        </div>
+        <section class="main-address">
+          <h3>Addis Ababa, Ethiopia</h3>
+          <p>@&nbsp;Akaki Kality sub city, Wereda 07, Kebele 02, Youth Hall</p>
```

Comment:
- [ ] This part of the code has to be improved: as the previous review mentioned, you need to make it responsive both Desktop and mobile, but on small screens, its display is not correct. You can see here that on small screens the text is hidden, only the `Addis Ababa` part is shown:
![2022-04-16 00 18 18](https://user-images.githubusercontent.com/49064106/163628520-0bb9730c-bca1-499d-bb97-514323be4855.gif)